### ISSUE TITLE
Why it export generated duplicated types

### ISSUE THREAD

**Issue by tianyingchun** [timocov/dts-bundle-generator#141]
**Bug report**
`dts` it export generated duplicated types 

**Input code**

```ts
// For `/class-names/index.ts` module
import { classNames } from './class-names';
export default classNames;

// For `/is-promise/index.ts` module
import { isPromise } from './is-promise';
export default isPromise;

// Extract all export into `/index.ts`
export { default as classNames } from './class-names';
export { default as isPromise } from './is-promise';

```

**Expected output**

```ts
export declare const classNames: ClassNamesFn;
export declare const isPromise: (value: any) => boolean;

export {
	classNames ,
	isPromise,
};

```

**Actual output**

```ts
export declare const classNames: ClassNamesFn;
export declare const isPromise: (value: any) => boolean;

export {
	classNames as classNames,
	isPromise as isPromise,
};

export {};
```

**Additional context**
Export declaration conflicts with exported declaration of 'classNames'.
Export declaration conflicts with exported declaration of 'isPromise'.

**Comment by tianyingchun** [comment]
BTW i don't want to change the export name `classNames `  to `aliasClassName`  for module`class-names/index.ts`.

**Comment by timocov** [comment]
I guess that the correct output should be something like this:

```ts
export declare type ClassNamesFn = () => void;
export declare const classNames: ClassNamesFn;
export declare const isPromise: (value: unknown) => boolean;

export {};
```

**Comment by timocov** [comment]
The fix is [just published in 5.5.0 version](https://github.com/timocov/dts-bundle-generator/releases/tag/v5.5.0).

**Comment by tianyingchun** [comment]
💯 it works fine `5.5.0`
